File: backend/hackathon-be/src/main/java/com/likelion/hackatonbe/domain/user/dto/LoginResponse.java
package com.likelion.hackatonbe.domain.user.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;

// LoginResponse.java
@Getter
@AllArgsConstructor
public class LoginResponse {
    private String token;
    private Long userId;

    @JsonProperty("isOnboarded") // 명시적 지정
    private boolean isOnboarded;

    private Long childId;
}

File: mobile/AtoCueMobile/app/src/main/java/com/example/atocuemobile/network/dto/LoginResponse.kt
package com.example.atocuemobile.network.dto

import com.google.gson.annotations.SerializedName

data class LoginResponse(
    val token: String,
    val userId: Long,
    @SerializedName("isOnboarded")
    val isOnboarded: Boolean,
    val childId: Long?
)